aspnetcore/data/ef-mvc/concurrency.md

```diff
@@ -151,37 +151,31 @@ The code in the catch block for that exception gets the affected Department enti
 
 [!code-csharp[Main](intro/samples/cu/Controllers/DepartmentsController.cs?range=164)]
 
-The `Entries` collection will have just one `EntityEntry`, object, and that object has the new values entered by the user.
+The `Entries` collection will have just one `EntityEntry` object, and you can use that object to get the new values entered by the user abd the current database values.
```

Comment:
consider. 
The `Entries` collection will have just one `EntityEntry` object.  You